code/modules/modular_computers/file_system/programs/civilian/cargo_control.dm

```diff
@@ -108,6 +104,14 @@
 		ui.set_auto_update(TRUE)
 
 /datum/nano_module/program/civilian/cargocontrol/Topic(href, href_list)
+	if(..())
+		return TRUE
+
+	var/mob/user = usr
+	if(!istype(user))
```

Comment:
`usr` is _always_ of type `mob`. The variable `user` is pointless and should probably be avoided.